### PR TITLE
Correct documentation w.r.t gitops repo creation.

### PR DESCRIPTION
**Pull request by iam-veeramalla**
**What type of PR is this?**
> /kind bug

**What does this PR do / why we need it**:
PR corrects the information provided in the docs

**Have you updated the necessary documentation?**
Yes

* [ ] Documentation has been updated.

**Which issue(s) this PR fixes**:
NA

**How to test changes / Special notes to the reviewer**:
Run `kam bootstrap` without creating a repo in github as suggested in documentation previously.

### CODE REVIEW

**Comment by amitkrout** [comment]
@iam-veeramalla Good catch. 

Currently i am stuck in CI failure https://github.com/redhat-developer/kam/issues/94 otherwise would have hit this error in my ongoing e2e test pr https://github.com/redhat-developer/kam/pull/87.

I will update my pr accordingly. thanks

File: docs/journey/day1/README.md [hunk 1]
```diff
@@ -19,7 +19,6 @@ You need to have the following installed in the OCP 4.x cluster.
 
 And, you will need this.
 
-* Create your [GitOps repository](prerequisites/gitops_repo.md) and leave it uninitialized.
```
Comment (amitkrout) [inline docs/journey/day1/README.md:22]: We don't need this preq anymore ?

Comment (iam-veeramalla) [reply docs/journey/day1/README.md:22]: I think we don't as  kam is creating the repo using authentication token.

**Review by bigkevmcd — APPROVED** [review]

**Pull request merged**